src/h5web/visualizations/line/DataCurve.tsx

```diff
@@ -30,10 +32,10 @@ function DataCurve(props: Props): JSX.Element {
     const abscissaScale = getAxisScale(abscissaInfo, width);
     const ordinateScale = getAxisScale(ordinateInfo, height);
 
-    const points = values.map((val, index) => {
+    const points = ordinates.map((val, index) => {
       const ordinate = ordinateScale(val);
       return new Vector3(
-        abscissaScale(index),
+        abscissaScale(abscissas[index]),
```

Comment:
As explained in https://github.com/silx-kit/h5web/pull/251/files#r506183326, the axis coordinates are now `abscissas` and not always indices. This means the curve construction must be matched.